runtime/lua/vim/snippet.lua

```diff
@@ -674,7 +674,9 @@ end
 --- @return boolean
 function M.active(filter)
   local active = M._session ~= nil and M._session.bufnr == vim.api.nvim_get_current_buf()
-
+  if M._session.current_tabstop and M._session.current_tabstop.index == 0 then
```

Comment:
when we have a reference, it's always worth a brief comment to provide that. it's a cheap way to help people reading this code in the future.

```suggestion
  -- "$0 defines the final tab stop".
  -- https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#completionClientCapabilities
  if M._session.current_tabstop and M._session.current_tabstop.index == 0 then
```

Reply:
because its too long.. exceeds the 80 column width 🤔